### DISCUSSION TITLE
Regarding "Definitions like #define SOME_VALUE 25 are not exported"

### DISCUSSION THREAD

**Discussion by faveoled** [indoorvivants/sn-bindgen#281] [Q&A]
https://sn-bindgen.indoorvivants.com/limitations/index.html#definitions-like-define-some-value-25-are-not-exported
Not sure how to implement this in my project. Should it be a *.c or *.h file (or both?)? How can I integrate those files into my Scala project build files?

I tried as *.h and then compiling the bindings, got this:
```
[error] /usr/bin/ld: /home/user/Dev/Scala/scala-native/target/scala-3.3.3/native/2.ll.o: in function `_SM25libmyconstants.functions$D12get_O_RDONLYiEO':
[error] 2.ll:(.text+0x9f42): undefined reference to `get_O_RDONLY'
```

**Comment by keynmol** [comment]
Hi @faveoled, I believe it should be a `.c` file - the example in the document is inspired by what I did for Twotm8: https://github.com/keynmol/twotm8/blob/main/bindings/src/main/resources/scala-native/my.c

And it should be put in `src/main/resources/scala-native/` for the Scala Native tooling to pick it up and compile alongside your other sources.
